ui/raidboss/data/06-ew/dungeon/alzadaals_legacy.ts

```diff
@@ -1,14 +1,43 @@
 import Conditions from '../../../../../resources/conditions';
+import Outputs from '../../../../../resources/outputs';
 import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  tentacleEffects: NetMatches['MapEffect'][];
+}
+
+export type Cardinal = 'N' | 'E' | 'S' | 'W';
+export type Intercard = 'NW' | 'NE' | 'SE' | 'SW';
+export const tentacleFlags = ['00080004', '00200004', '00800004', '02000004'];
+export type TentacleFlag = typeof tentacleFlags[number];
```

Comment:
This is just a string, I think.  (You have to say `export const tentacleFlags etc etc as const` to have a non-string type if you wanted to, but that will cause issues elsewhere in places that want strings.)

In other words, I think it's fine that `tentacleFlags` is a `string[]`, and we can probably drop `TentacleFlag` as an explicit type, and also just say `key: string` instead of `key in TentacleFlag` below.  It's not a logic change, but just a way to be more readable.